src/contracts/core/StrategyManager.sol

```diff
@@ -290,16 +290,17 @@ contract StrategyManager is
         uint256[] calldata strategyIndexes,
         IStrategy[] calldata strategies,
         uint256[] calldata shares,
-        address withdrawer,
-        bool undelegateIfPossible
+        address withdrawer
```

Comment:
I realize we agreed on this change, but any concerns about changing this function signature?
I guess we can still support the same function signature with a second method that just ignores the bool parameter, if we decide it's necessary

Reply:
From my perspective, it's not a big deal - but I guess frontend or someone else might have opinions? idk who might be relying on it

Reply:
Yeah, this could be a frontend issue. I'll can make a quick commit that should give us the ability to call with consistent function signature, and explain in a comment

Reply:
actually, working through this, it might be a pain to maintain both versions of the function. @gpsanant any estimate on the pain the frontend team would have dealing with a change in function signature here?

Reply:
discussed offline. This is definitely easier to coordinate one time on the frontend than keep maintaining two parallel functions, so I'm just going to close this thread out.